cryptoinvestor/models/user.py

```diff
@@ -7,3 +8,44 @@ def __init__(self, *, username: str, password: str, account: Account):
         self.username = username
         self.password = password
         self.account = account
+        self.id = id
+        self.rates = {}
+        self.coin_status = 10000
+        self.bought = dict()
+
+    def buy(self) -> bool:
+        count = request.args.get('count')
+        count = float(count)
+        crypto_id = request.args.get('id')
+        price = request.args.get('rate')
+        price = float(price)
+        crypto_name = request.args.get('name')
```

Comment:
Why do you need to pass both id and name of the currency? We have a list of all loaded currencies in application.

Reply:
Crypto_id was unused.... removed